src/main/java/uk/gov/hmcts/reform/sscs/HearingsApiApplication.java

```diff
@@ -1,20 +1,86 @@
 package uk.gov.hmcts.reform.sscs;
 
+import com.microsoft.applicationinsights.web.internal.ApplicationInsightsServletContextListener;
+import okhttp3.OkHttpClient;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.web.servlet.ServletListenerRegistrationBean;
 import org.springframework.cloud.openfeign.EnableFeignClients;
+import org.springframework.context.MessageSource;
+import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.support.ReloadableResourceBundleMessageSource;
+import org.springframework.scheduling.annotation.EnableScheduling;
+import org.springframework.web.client.RestTemplate;
+import uk.gov.hmcts.reform.sscs.ccd.config.CcdRequestDetails;
+
+import java.util.concurrent.TimeUnit;
+import javax.servlet.ServletContextListener;
```

Comment:
Would it better to use Jakarta instead of Javax to stay up to date with the latest?